packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -730,30 +730,18 @@ export function fromShellExecution(execution: theia.ShellExecution, processTaskD
     }
 }
 
-export function getProcessExecution(processTaskDto: ProcessTaskDto): theia.ProcessExecution {
-    const execution = {} as theia.ProcessExecution;
-
-    execution.process = processTaskDto.command;
-
-    const processArgs = processTaskDto.args;
-    execution.args = processArgs ? processArgs : [];
-
-    const options = processTaskDto.options;
-    execution.options = options ? options : {};
-
-    return execution;
-}
-
-export function getShellExecution(processTaskDto: ProcessTaskDto): theia.ShellExecution {
-    const execution = {} as theia.ShellExecution;
-
-    const options = processTaskDto.options;
-    execution.options = options ? options : {};
-    execution.args = processTaskDto.args;
-
-    execution.command = processTaskDto.command;
-
-    return execution;
+export function getProcessExecution(taskDto: TaskDto): theia.ProcessExecution {
```

Comment:
The function takes `TaskDto` now, so unnecessary assertion in [the place of call](https://github.com/eclipse-theia/theia/blob/610d721c30d5c4892c673455cc350be82e9ddd82/packages/plugin-ext/src/plugin/type-converters.ts#L672)